src/sentry/templates/sentry/login.html

```diff
@@ -117,6 +125,35 @@ <h3>{% trans "Sign in to continue" %}</h3>
             </div>
           </form>
         </div>
+        {% if request.get_host == "pri-sentry.sentry.io" %}
```

Comment:
this will be removed and it is here so then I can test it before releasing it to everyone. There is no way to test this using flagpole as the user is not logged yet